src/main/java/seedu/address/model/person/IsArchivePredicate.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person;
```

Comment:
I think its a little strange to put a predicate in the person folder. Perhaps we can package it in model with where it is used at or put the whole class into the Model class.

Reply:
The predicate is meant for filtering the field in the Person, just like how we placed the Predicate for filtering Name in the same folder